webapp/components/comments/EditCommentForm/index.vue

```diff
@@ -0,0 +1,121 @@
+<template>
+  <ds-form v-model="form" @submit="handleSubmit">
+    <template slot-scope="{ errors }">
+      <ds-card>
+        <!-- <no-ssr> -->
+        <hc-editor ref="editor" :users="users" :value="form.content" @input="updateEditorContent"/>
+        <!-- </no-ssr> -->
```

Comment:
I think we need the `no-ssr`, I was interacting with the `ContributionForm` yesterday and if you refresh the page with these commented out, it throws an error about `document`, which is related to Service-Side rendering